cognite/client/data_classes/transformations/__init__.py

```diff
@@ -381,6 +401,34 @@ class TransformationList(CogniteResourceList):
     _RESOURCE = Transformation
 
 
+class TagsFilter:
+    @abstractmethod
+    def dump(self) -> Dict[str, Any]:
+        ...
```

Comment:
this will possibly be extended with `containsAll` later.

Reply:
Nice approach!